src/adapters/http/server.ts

```diff
@@ -0,0 +1,132 @@
+import Adapter from "../../lib/adapter.js"
+import GleeMessage from "../../lib/message.js"
+import http from "http"
+import { validateData } from "../../lib/util.js"
+import GleeError from "../../errors/glee-error.js"
+import * as url from "url"
+
+class HttpAdapter extends Adapter {
+  private res = new Map()
+
+  name(): string {
+    return "HTTP server"
+  }
+
+  async connect(): Promise<this> {
+    return this._connect()
+  }
+
+  async send(message: GleeMessage): Promise<void> {
+    return this._send(message)
+  }
+
+  async _connect(): Promise<this> { // NOSONAR
```

Comment:
Please, move this logic to the `connect` method instead. Also, why `NOSONAR` here?